build/de.cau.cs.kieler.klighd.targetplatform/pom.xml

```diff
@@ -3,7 +3,7 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>de.cau.cs.kieler.klighd.targetplatform</artifactId>
+  <artifactId>de.cau.cs.kieler.klighd.targetplatform-elkNightly</artifactId>
```

Comment:
Should not be part of the master

Reply:
Can be resolved once ELK 0.7.0 is released.